### ISSUE TITLE
Dispatch throws errors

### ISSUE THREAD

**Issue by mark2185** [ilyachur/cmake4vim#93]
**Describe the bug**
Using `g:cmake_build_executor='dispatch'` produces error about `missing endif` in `Dispatch` code.

**To Reproduce**
1. Use `dispatch` with anything.

**Expected behavior**
It should just work.

**Desktop (please complete the following information):**
 - OS: macOS
 - Editor: vim
 - Version: 8.2

**Additional context**
I'll look into it tomorrow, I'm just creating an issue to help if someone steps on this bug in the meantime.

**Comment by ilyachur** [comment]
@mark2185 Could you provide an additional information?
I always use `Dispatch` and it works as expected on my machine.... But it looks terrible and can affect a lot of users.

**Comment by mark2185** [comment]
Well it seems to be a dispatch error of some sort, I don't know what triggered it. 
I thought it was related to the `quickfix_title` change from `ctest-args` PR, that's why I opened the issue before going to sleep last night. But it's not, I tried reverting it, I tried checking out previous commits of `dispatch`, but it didn't help.

Anyway, I'll bring report this to `vim-dispatch` and in the meantime I'll try to fix the `job` executor output.
